variables.tf

```
@@ -365,3 +359,16 @@ YAML
     enable = false
   }
 }
+
+variable "aws_load_balancer_controller_config" {
+  type = object({
+    enable          = optional(bool, false)
+    helm_repository = optional(string, "https://aws.github.io/eks-charts")
+    helm_version    = optional(string, "1.4.5")
+    chart_values    = optional(string, false)
```

Should we add an empty yaml here as in the case of the cluster autoscaler:
```suggestion
    chart_values    = optional(string, <<-YAML
    
    YAML
    )
```